Jenkinsfile-container

```diff
@@ -0,0 +1,44 @@
+pipeline {
+	agent none
+
+	options {
+		disableConcurrentBuilds()
+		buildDiscarder(logRotator(numToKeepStr: '14'))
+	}
+
+	stages {
+		stage('Build the Spring Data release tools container') {
+			when {
+				branch 'container'
```

Comment:
Let's scope the conditions to a changed `Dockerfile` only and ignore the `branch`